cmd/manager/operator.go

```diff
@@ -59,7 +59,11 @@ var (
 		"rhcos4",
 		"ocp4",
 	}
+)
+
+const (
 	defaultScanSettingsName = "default"
+	applyScanSettingsName   = "apply"
```

Comment:
I was thinking since this is just a variant of default, it might be clearer as `default-apply` or `default-auto-apply`